Dockerfile

```diff
@@ -450,6 +451,12 @@ RUN echo "BUILD_BRANCH=${BUILD_BRANCH}"    | tee /build.info \
     && echo "FDD_LENSES_CLI_VERSION=${LC_VERSION}" | tee -a /build.info \
     && sed -e 's/^/FDD_/' /opt/landoop/build.info  | tee -a /build.info
 
+# Add man pages & README.txt
+ADD ./filesystem/usr/local/man /usr/local/share/
+ADD ./filesystem/usr/local/README.txt /README.txt
+RUN ln -sf "/usr/local/share/man" \
```

Comment:
Better instead of this, add to `filesystem/usr/local/share/landoop/etc/bashrc` this:
```export MANPATH=/usr/share/man:/usr/local/share/man```

Because if the user ever does `apk add --no-cache man-pages` with this trick it will work. With yours, it will break. :)

Reply:
Moved it under /usr/local which is a default manpath.